std_msgs/Header header
float64 cam_y_dist_reader        # cam_y_dist_reader reading
float64 cam_y_dist_reader_var    # cam_y_dist_reader_var reading


